workspace/coordinates.ts

```diff
@@ -0,0 +1,32 @@
+const request = require('request')
+
+import { BusStop } from './busStop'
+
+export class Coordinates {
+    latitude: string;
+    longitude: string;
+
+    constructor(lat: string, lon: string){
+        this.latitude = lat; 
+        this.longitude = lon;
+    }
```

Comment:
You can use your `public lat: string, etc.` stuff from the other constructors to cut some of the boilerplate here.